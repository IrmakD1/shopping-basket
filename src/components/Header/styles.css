.header {
    padding: 20px;
    background: #5171b6;
    text-align: center;
    color: aliceblue;
    margin-bottom: 1.25rem;
  }