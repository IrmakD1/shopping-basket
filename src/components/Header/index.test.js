import React from 'react';
import renderer from 'react-test-renderer';
import Header from '.';

describe('Header Component', () => {
    let component;
    let tree;
  
    beforeEach(() => {
      component = renderer.create(
        <div>
          <Header text='I am a Header'/>
        </div>
      );
      tree = component.toJSON();
    });
  
    test('it should render', () => {
      expect(tree).toMatchSnapshot();
    });
  });
  