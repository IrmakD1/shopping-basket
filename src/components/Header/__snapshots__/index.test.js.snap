// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component it should render 1`] = `
<div>
  <div
    className="header"
  >
    <h1>
      I am a Header
    </h1>
  </div>
</div>
`;
